features/util.go

```diff
@@ -141,9 +148,17 @@ func (b *featureGateBuilder) register() (configv1.FeatureGateName, error) {
 	}
 
 	featureGateName := configv1.FeatureGateName(b.name)
+	var minComponentVersions []configv1.RequiredMinimumComponentVersion
+	if b.minimumKubeletVersion != "" {
+		minComponentVersions = []configv1.RequiredMinimumComponentVersion{{
+			Component: configv1.RequiredMinimumComponentKubelet,
+			Version:   b.minimumKubeletVersion,
+		}}
```

Comment:
If you were to append to the existing slice here, this would age better as we add future minimum versions

Reply:
done!